locations/models.py

```diff
@@ -160,6 +162,12 @@ def main_image(self):
         )
     ]
 
+    def get_context(self, request):
+        context = super().get_context(request)
+        locationtagindexset=LocationTagIndexPage.objects.live().sibling_of(self)
+        context["tagspage"]=locationtagindexset[0]
```

Comment:
this could probably be: `context['tagspage'] = LocationTagIndexPage.objects.live().sibling_of(self).first()`

Reply:
or probably better: `context['tagspage'] = LocationTagIndexPage.objects.live().child_of(LocationsIndexPage.objects.live().ancestor_of(self).first()).first()`